catalog/mapbiomas-public/projects_mapbiomas-public_assets_brazil_lulc_v1.jsonnet

```diff
@@ -0,0 +1,201 @@
+              'd4271e',  // [24] Urban Area
+              'db4d4f',  // [25] Other Non Vegetated Areas
+              '0000FF',  // [26] --
+              '000000',  // [27] --
+              '000000',  // [28] --
+              'ffaa5f',  // [29] Rocky Outcrop
+              '9c0027',  // [30] Mining
+              '091077',  // [31] Aquaculture
+              'fc8114',  // [32] Hypersaline Tidal Flat
+              '2532e4',  // [33] Rivers, Lakes and Ocean
+              '93dfe6',  // [34] Glacier
+              '9065d0',  // [35] --
+              'd082de',  // [36] --
+              '000000',  // [37] --
+              '000000',  // [38] --
+              'f5b3c8',  // [39] Soybean
+              'c71585',  // [40] Rice
+              'f54ca9',  // [41] Other Temporary Crops
+              'cca0d4',  // [42] --
+              'dbd26b',  // [43] --
+              '807a40',  // [44] --
+              'e04cfa',  // [45] --
+              'd68fe2',  // [46] Coffee
+              '9932cc',  // [47] Citrus
+              'e6ccff',  // [48] Other Perennial Crops
+              '02d659',  // [49] Wooded Sandbank Vegetation
+              'ad5100',  // [50] Herbaceous Sandbank Vegetation
+              '000000',  // [51] --
+              '000000',  // [52] --
+              '000000',  // [53] --
+              '000000',  // [54] --
+              '000000',  // [55] --
+              '000000',  // [56] --
+              'CC66FF',  // [57] --
+              'FF6666',  // [58] --
+              '006400',  // [59] --
+              '8d9e8b',  // [60] --
+              'f5d5d5',  // [61] Salt Flats
+              'ff69b4',  // [62] Cotton
+              'ebf8b5',  // [63] --
+              '000000',  // [64] --
+              '000000',  // [65] --
+              '91ff36',  // [66] --
+              '7dc975',  // [67] --
+              'e97a7a',  // [68] --
+              '0fffe3',  // [69] Coral Reefs
+              '000000',  // [70] --
+              '000000',  // [71] --
+              '000000',  // [72] --
+              '000000',  // [73] --
+              '000000',  // [74] --
+              'c12100',  // [75] Photovoltaic Power Plant
+            ],
+          },
+        },
+      },
+    ],
+
+  },
+
+  'sci:doi': '10.3390/rs12172735',
+  'sci:citation': |||
+    Souza et al. (2020) – Reconstructing Three Decades of Land Use and Land Cover Changes in Brazilian Biomes with Landsat Archive and Earth Engine – Remote Sensing, 
+    Volume 12, Issue 17, 10.3390/rs12172735.
+
+    [doi:10.3390/rs12172735](https://doi.org/10.3390/rs12172735)
+
+  |||,
+
+  'gee:terms_of_use': ee.gee_terms_of_use(license),
+
+  'gee:status': 'beta',
+
+  'gee:type': ee_const.gee_type.image_collection,
+
+  license: license.id,
+
+  links: ee.standardLinks(subdir, id),
+
+  type: ee_const.stac_type.collection,
+  stac_version: ee_const.stac_version,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
```

Comment:
could you please also add a gee:schema section (like in https://github.com/google/earthengine-catalog/blob/main/catalog/ESA/ESA_WorldCereal_2021_MARKERS_v100.jsonnet#L97) describing all the properties that might be different. Looks like this means "year", "version", and "collection id"?